docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -24,10 +32,41 @@ To create a new user:
 4. Review the new user information and note the temporary password that has been assigned. To send the user an introductory email, select the checkbox.
 5. Press *Done* when complete.
 
+## Users Table
+
+The *Users* table displays a list of users who are authorized to login to the Run:ai platform.
+
+!!! Note
+    SSO users who have not logged into the Run:ai platform will not appear in the table until they do.
+
+The *Users* table contains informtion about users who have logged into the Run:ai Platform. The table includes columns for:
```

Comment:
@jasonnovichRunAI this is not accurate. SSO users will be displayed once they log in (as you also wrote in the note in line 40, but local users are displayed once you create them.
So maybe we should change the sentence to "The users table contains information about users in the Run:ai platform"?